redbot/cogs/mutes/mutes.py

```diff
@@ -640,7 +656,13 @@ async def on_guild_channel_update(
                     )
                     log.debug("created case")
                     if await self.config.guild(after.guild).dm():
-                        await self._send_dm_notification(modlog_case)
+                        await self._send_dm_notification(
+                            after,
+                            None,
+                            after.guild,
+                            _("(Voice-)Channel unmute"),
```

Comment:
This should differ between voice and channel mute, there's even code checking for the channel type just before creating the case in here, so you could choose a proper string there.